diff -Nrup xkeyboard-config-1.7-patched/keycodes/evdev xkeyboard-config-working/keycodes/evdev
--- xkeyboard-config-1.7-patched/keycodes/evdev	2010-02-11 12:27:43.000000000 -0800
+++ xkeyboard-config-working/keycodes/evdev	2010-02-11 12:30:04.000000000 -0800
@@ -189,6 +189,7 @@ default xkb_keycodes "evdev" {
 	<LNFD> = 109;	// #define KEY_LINEFEED            101
 	<I120> = 120;	// #define KEY_MACRO               112
 	<I126> = 126;	// #define KEY_KPPLUSMINUS         118
+	<I128> = 128;   // #define KEY_SCALE               120
 	<I129> = 129;	// #define KEY_KPCOMMA             121
 	<I147> = 147;	// #define KEY_MENU                139
 	<I148> = 148;	// #define KEY_CALC                140
@@ -238,6 +239,7 @@ default xkb_keycodes "evdev" {
 	<I209> = 209;	// #define KEY_PAUSECD             201
 	<I210> = 210;	// #define KEY_PROG3               202
 	<I211> = 211;	// #define KEY_PROG4               203 conflicts with AB11
+	<I212> = 212;   // #define KEY_DASHBOARD           204
 	<I213> = 213;	// #define KEY_SUSPEND             205
 	<I214> = 214;	// #define KEY_CLOSE               206
 	<I215> = 215;	// #define KEY_PLAY                207
diff -Nrup xkeyboard-config-1.7-patched/symbols/inet xkeyboard-config-working/symbols/inet
--- xkeyboard-config-1.7-patched/symbols/inet	2010-02-11 12:27:43.000000000 -0800
+++ xkeyboard-config-working/symbols/inet	2010-02-11 12:30:16.000000000 -0800
@@ -123,6 +123,7 @@ xkb_symbols "evdev" {
 
 //  key <I120>   {      [ ]       }; // KEY_MACRO 
     key <I126>   {      [ plusminus             ]       };
+    key <I128>   {      [ XF86LaunchA           ]       };
     key <I129>   {      [ KP_Decimal            ]       };
     key <I147>   {      [ XF86MenuKB            ]       };
     key <I148>   {      [ XF86Calculator        ]       };
@@ -172,6 +173,7 @@ xkb_symbols "evdev" {
     key <I209>   {      [ XF86AudioPause        ]       };
     key <I210>   {      [ XF86Launch3           ]       };
     key <I211>   {      [ XF86Launch4           ]       };
+    key <I212>   {      [ XF86LaunchB           ]       };
     key <I213>   {      [ XF86Suspend           ]       };
     key <I214>   {      [ XF86Close             ]       };
     key <I215>   {      [ XF86AudioPlay         ]       };
