diff --git a/rules/base.lists.part b/rules/base.lists.part
index d1927c5..69de1f6 100644
--- a/rules/base.lists.part
+++ b/rules/base.lists.part
@@ -67,7 +67,8 @@
               sk1300 sk2500 sk6200 sk7100 \
               sven sven303 symplon \
               tm2030USB toshiba_s3000 trust trustda \
-              unitekkb1925 yahoo
+              unitekkb1925 yahoo \
+              intelligent_keyboard_k04
 
 // all layouts with 3rd and 4th groups
 ! $threelevellayouts = al az \
diff --git a/rules/base.xml.in b/rules/base.xml.in
index f43e651..4038bcd 100644
--- a/rules/base.xml.in
+++ b/rules/base.xml.in
@@ -1196,6 +1196,13 @@
         <vendor>Creative</vendor>
       </configItem>
     </model>
+    <model>
+      <configItem>
+        <name>intelligent_keyboard_k04</name>
+        <description>Intelligent Keyboard K04</description>
+        <vendor>Generic</vendor>
+      </configItem>
+    </model>
   </modelList>
   <layoutList>
     <layout>
diff --git a/symbols/inet b/symbols/inet
index f6cd6ac..e9b5a59 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -1821,3 +1821,30 @@ partial alphanumeric_keys
 xkb_symbols "pc105" {
     include "inet(media_nav_acpi_common)"
 };
+
+//Intelligent Keyboard
+
+//Intelligent Keyboard K04
+partial alphanumeric_keys
+xkb_symbols "intelligent_keyboard_k04" {
+    key <I22> { [ XF86AudioPlay, XF86AudioPause ] };
+    key <I10> { [ XF86AudioPrev                 ] };
+    key <I24> { [ XF86AudioStop                 ] };
+    key <I19> { [ XF86AudioNext                 ] };
+    key <I6D> { [ XF86AudioMedia                ] };
+    key <I30> { [ XF86AudioRaiseVolume          ] };
+    key <I20> { [ XF86AudioMute                 ] };
+    key <I2E> { [ XF86AudioLowerVolume          ] };
+    key <I6A> { [ XF86Back                      ] };
+    key <I69> { [ XF86Forward                   ] };
+    key <I68> { [ XF86Stop                      ] };
+    key <I67> { [ XF86Refresh                   ] };
+    key <I65> { [ XF86Search                    ] };
+    key <I66> { [ XF86Favorites                 ] };
+    key <I32> { [ XF86HomePage                  ] };
+    key <I6C> { [ XF86Mail                      ] };
+    key <I63> { [ XF86WakeUp                    ] };
+    key <I6B> { [ XF86MyComputer                ] };
+    key <I21> { [ XF86Calculator                ] };
+    key <I5F> { [ XF86Sleep                     ] };
+};
