Subject: Correct spelling error from 3d to 3rd
Origin: other, https://bugs.launchpad.net/ubuntu/+source/xkeyboard-config/+bug/398873
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/xkeyboard-config/+bug/398873
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=26528
Author: Nigel Babu <nigelbabu@gmail.com>

Index: xkeyboard-config-1.7/rules/base.xml.in
===================================================================
--- xkeyboard-config-1.7.orig/rules/base.xml.in	2010-02-16 10:06:25.772010161 +0530
+++ xkeyboard-config-1.7/rules/base.xml.in	2010-02-16 10:06:32.336010209 +0530
@@ -4997,7 +4997,7 @@
       <option>
         <configItem>
           <name>numpad:shift3</name>
-          <_description>Shift does not cancel NumLock, chooses 3d level instead</_description>
+          <_description>Shift does not cancel NumLock, chooses 3rd level instead</_description>
         </configItem>
       </option>
       <option>
