commit d33ce57e8411b5cd63070086e56229d1839a0ff4
Author: Sergey V. Udaltsov <svu@gnome.org>
Date:   Thu Jan 28 23:52:09 2010 +0000

    slightly improved variant names

diff --git a/rules/base.xml.in b/rules/base.xml.in
index 8ec086e..76b0b98 100644
--- a/rules/base.xml.in
+++ b/rules/base.xml.in
@@ -4344,19 +4344,19 @@
       <variantList>
         <variant>
           <configItem>
-            <name>fross</name>
+            <name>fr-oss</name>
             <description>Français (France Alternative)</description>
           </configItem>
         </variant>
         <variant>
           <configItem>
-            <name>usmac</name>
+            <name>us-mac</name>
             <description>English (USA Macintosh)</description>
           </configItem>
         </variant>
         <variant>
           <configItem>
-            <name>usintl</name>
+            <name>us-intl</name>
             <description>English (USA International)</description>
           </configItem>
         </variant>
diff --git a/symbols/ml b/symbols/ml
index 732eab3..2ca4384 100644
--- a/symbols/ml
+++ b/symbols/ml
@@ -17,7 +17,7 @@ xkb_symbols "basic" {
 };
 
 partial alphanumeric_keys
-xkb_symbols "fross" {
+xkb_symbols "fr-oss" {
 
     include "fr(oss)"
 
@@ -39,7 +39,7 @@ xkb_symbols "fross" {
 };
 
 partial alphanumeric_keys
-xkb_symbols "usmac" {
+xkb_symbols "us-mac" {
 
     include "us(mac)"
 
@@ -62,7 +62,7 @@ xkb_symbols "usmac" {
 };
 
 partial alphanumeric_keys
-xkb_symbols "usintl" {
+xkb_symbols "us-intl" {
 
     include "us(intl)"
 
